images/linux/scripts/installers/java-tools.sh

```diff
@@ -61,28 +62,23 @@ chmod -R 777 /usr/lib/jvm
 apt-fast install -y --no-install-recommends ant ant-optional
 echo "ANT_HOME=/usr/share/ant" | tee -a /etc/environment
 
-# Install Maven
-curl -sL https://www-eu.apache.org/dist/maven/maven-3/3.8.1/binaries/apache-maven-3.8.1-bin.zip -o maven.zip
-unzip -qq -d /usr/share maven.zip
-rm maven.zip
+# Install Maven 3.8.1
+mavenDownloadUrl="https://www-eu.apache.org/dist/maven/maven-3/3.8.1/binaries/apache-maven-3.8.1-bin.zip"
```

Comment:
maybe move maven version to the variable, so when we need to update it - we will have to change version in one place only?

Reply:
Sounds good. I hope they won't change the download server along with the version 🤞